zeebe/gateway-protocol/vacuum-ruleset.yaml

```diff
@@ -23,12 +21,30 @@ rules:
     type: validation
   require-property-descriptions:
     category:
-      description: Documentation is really important, in OpenAPI, just about everything can and should have a description. This set of rules checks for absent descriptions, poor quality descriptions (copy/paste), or short descriptions.
       id: descriptions
-      name: Descriptions
     description: Schema properties must have a description.
     given: $.components.schemas[*].properties[*]
     then:
       field: description
       function: truthy
     severity: error
+  operation-key-properties-must-be-strings:
+    category:
+      id: operations
+    id: operation-key-properties-must-be-strings
+    description: "`...Key` parameters must be of type `string`."
```

Comment:
Not actually used; the description is coming from the rule itself. I could delete this, or I could update the rule to use the description passed in.

Reply:
I think this description is just for meta level, for humans looking at the rule to understand what this does and checks.